tests/unit/reports/single_table/_properties/test_synthesis.py

```diff
@@ -0,0 +1,94 @@
+from unittest.mock import Mock, call, patch
+
+import pandas as pd
+
+from sdmetrics.reports.single_table._properties.synthesis import Synthesis
+
+
+class TestSynthesis:
+
+    @patch('sdmetrics.reports.single_table._properties.synthesis.'
+           'NewRowSynthesis.compute_breakdown')
+    def test__generate_details(self, newrowsynthesis_mock):
+        """Test the ``_generate_details`` method."""
+        # Setup
+        real_data = pd.DataFrame({
```

Comment:
I don't think you need the real_data/metadata block, since they are not used for anything (just pass empty dataframes or mocks), and synthetic_data can just be a one liner, since it only checks the size of that. Just to make it more obvious that what's actually been tested is the return of the mock.

Reply:
Yes, done in [b00d7ca](https://github.com/sdv-dev/SDMetrics/pull/398/commits/b00d7cafd54bf24f3b76c78763bddbd65d5b9408)